systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -373,4 +373,14 @@ public static boolean isParallelNamespaceTest(ExtensionContext extensionContext)
                 .toLowerCase(Locale.ENGLISH)
                 .contains(PARALLEL_NAMESPACE)).count() == 1;
     }
+
+    /**
+     * Retrieve namespace based on the cluster configuration
+     * @param singleNamespace suite namespace
+     * @param extensionContext test context for get the parallel namespace
+     * @return single or parallel namespace based on cluster configuration
+     */
+    public static String getNamespaceBasedOnRbac(String singleNamespace, ExtensionContext extensionContext) {
```

Comment:
```suggestion
    public static String getNamespaceBasedOnRbac(String coNamespace, ExtensionContext extensionContext) {
```

or maybe just a simple `namespace` ?